notebooks/data-sources/TestGrid/metrics/pct_fixed_each_ts.ipynb

```diff
@@ -2,20 +2,24 @@
  "cells": [
```

Comment:
<p>This looks like its also saving something to disk. This should be handled in the cells below. </p><p> </p><p>Also, here you are saving something called, <code>pct_fixed_per_ts</code>  and in the next cell you are saving <code>pct_fixed_per_ts_df</code>  this section should save all outputs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>